README.md

```diff
@@ -9,7 +9,7 @@ Currently, the following analyzers are included:
 
 - DHCP <sup>[1]</sup>
 - DNS <sup>[1]</sup>
-- [Facefish Rootkit](https://github.com/zeek/spicy-analyzers/tree/main/analyzer/facefish_rootkit)
+- Facefish Rootkit deprecated by <https://github.com/corelight/zeek-spicy-facefish>
```

Comment:
Let's instead pull this in as a dependency, see comments in https://github.com/zeek/spicy-analyzers/pull/102#pullrequestreview-815950402.